zeg/http.py

```diff
@@ -41,43 +41,47 @@ def make_session(endpoint, token):
     return session
 
 
+def handle_response(response):
+    if response.status_code == 400:
```

Comment:
I'd suggest something closer to the logic in our ClientError exception object, rather than special casing one code.